R/csv.R

```diff
@@ -100,6 +119,55 @@ duckdb_read_csv <- function(
   invisible(out)
 }
 
+
+#' Column names and types logic for duckdb_read_csv()
+#'
+#' @param found the detected (found) header and types from `utils::read_csv`
+#' @param col.names user provided column names
+#' @param col.types user provider column types and maybe names too
+#'
```

Comment:
```suggestion
#'
#' @noRd
```

To avoid generation of `.Rd` file.